components/Layout/Header.jsx

```diff
@@ -0,0 +1,29 @@
+import Link from "next/link";
+import Wrapper from "../Wrapper";
+import LinkItem from "../ui/LinkItem";
+
+const Header = () => {
+  return (
+    <Wrapper>
+      <div className="flex flex-col justify-between gap-5 py-12 text-xl lg:flex-row sm:text-2xl -tracking-tighter sm:tracking-normal">
+        <div>Ape Unit</div>
```

Comment:
The *Ape Unit* should be a heading for SEO (Search Engine Optimization)
```suggestion
        <h1>Ape Unit</h1>
```